Directory.Build.props

```diff
@@ -43,6 +43,7 @@
   <PropertyGroup>
     <!-- System packages -->
     <SystemDiagnosticsPerformanceCounterVersion>5.0.0</SystemDiagnosticsPerformanceCounterVersion>
+    <SystemDiagnosticsDiagnosticSourceVersion>6.0.0</SystemDiagnosticsDiagnosticSourceVersion>
```

Comment:
I think you should upgrade all libs (Ms.Extensions.*, and system packages) to 6.0 version to have consistent transitive references to DiagnosticSource and its dependencies.

Reply:
A commit(https://github.com/dotnet/orleans/commit/1460dce7901415e4c5f39e4c0f677b46b8c54c64) in another branch already did this, I assume that commit will be cherry-picked into main branch, @ReubenBond what do you think?

Reply:
Yes, we don't need to update it here